app/script/components/deviceCard.js

```diff
@@ -41,7 +41,7 @@ z.components.DeviceCard = class DeviceCard {
       this.data_uie_name += '-current';
     }
 
-    this.activated_in = ko.observable();
+    this.activated_in = ko.observable(z.l10n.text(z.string.preferences_devices_activated_in));
```

Comment:
Doesn't this need to be an observableArray with a default value of an array as well?